sdk/monitor/monitor-opentelemetry/src/shared/logging/logger.ts

```diff
@@ -61,8 +61,8 @@ export class Logger {
     });
 
     let azureLogLevelEnv =
-      process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL || process.env.AZURE_LOG_LEVEL;
-    let azureLogLevel: AzureLogLevel = "warning"; // default
+      process.env.APPLICATIONINSIGHTS_INSTRUMENTATION_LOGGING_LEVEL;
+    let azureLogLevel: AzureLogLevel = process.env.AZURE_LOG_LEVEL || "warning"; // default
```

Comment:
could we use a default case in the switch to do this assignment? Feels weird to have the default outside the control flow

Reply:
sure. Done that. Please check again :-)